po/it.po

```diff
@@ -1,5 +1,671 @@
+msgstr "Impostazioni"
+
+# ?
+#: Layouts/HeaderBar.vala:118
+msgid "Difference"
+msgstr "Differenza"
+
+#: Layouts/HeaderBar.vala:120
+msgid "Exclusion"
+msgstr "Esclusione"
+
+#: Layouts/HeaderBar.vala:122
+msgid "Intersect"
+msgstr "Intersezione"
+
+#: Layouts/HeaderBar.vala:124
+msgid "Union"
+msgstr "Unione"
+
+#: Layouts/HeaderBar.vala:161
+msgid "New Window"
+msgstr "Nuova finestra"
+
+#: Layouts/HeaderBar.vala:182
+msgid "Open Recent"
+msgstr "Apri recenti"
+
+#: Layouts/HeaderBar.vala:195
+msgid "Save As"
+msgstr "Salva come"
+
+#: Layouts/HeaderBar.vala:204
+msgid "Quit"
+msgstr "Esci"
+
+#: Layouts/HeaderBar.vala:237
+msgid "Artboard"
+msgstr "Artboard"
+
+#: Layouts/HeaderBar.vala:254
+msgid "Add Items"
+msgstr "Aggiungi elementi"
+
+#: Layouts/HeaderBar.vala:262
+msgid "Rectangle"
+msgstr "Rettangolo"
+
+#: Layouts/HeaderBar.vala:268
+msgid "Ellipse"
+msgstr "Ellisse"
+
+#: Layouts/HeaderBar.vala:286
+msgid "Vector"
+msgstr "Vettore"
+
+#: Layouts/HeaderBar.vala:288
+msgid "Pencil"
+msgstr "Matita"
+
+#: Layouts/HeaderBar.vala:291
+msgid "Text"
+msgstr "Testo"
+
+#: Layouts/HeaderBar.vala:297
+msgid "Image"
+msgstr "Immagine"
+
+#: Layouts/HeaderBar.vala:330
+msgid "Export Current Selection"
+msgstr "Esporta selezione corrente"
+
+#: Layouts/HeaderBar.vala:336
+msgid "Export Artboards"
+msgstr "Esporta le artboard"
+
+#: Layouts/HeaderBar.vala:345
+msgid "Highlight Area to Export"
+msgstr "Evidenza l'area da esportare"
+
+#: Layouts/HeaderBar.vala:391
+msgid "Main Menu"
+msgstr "Menu principale"
+
+#: Layouts/HeaderBar.vala:468 Services/ActionManager.vala:218
+#: Services/ActionManager.vala:237 Services/ActionManager.vala:256
+#, c-format
+msgid "Unable to open file at '%s'"
+msgstr "Non è possibile aprirle il file '%s'"
+
+#: Layouts/MainCanvas.vala:52
+msgid "Button was pressed!"
+msgstr "Il pulsante è stato premuto!"
+
+#: Layouts/MainCanvas.vala:160
+msgid "Export completed!"
+msgstr "Esportazione completata!"
+
+#: Layouts/Partials/AlignItemsPanel.vala:48
+msgid "Distribute Horizontally"
+msgstr "Distribuisci orizzontalmente"
+
+#: Layouts/Partials/AlignItemsPanel.vala:50
+msgid "Distribute Vertically"
+msgstr "Distribuisci verticalmente"
+
+#: Layouts/Partials/AlignItemsPanel.vala:52
+msgid "Align Left"
+msgstr "Allineamento a sinistra"
+
+#: Layouts/Partials/AlignItemsPanel.vala:53
+msgid "Align Center"
+msgstr "Allineamento al centro"
+
+#: Layouts/Partials/AlignItemsPanel.vala:54
+msgid "Align Right"
+msgstr "Allicenamento a destra"
+
+#: Layouts/Partials/AlignItemsPanel.vala:56
+msgid "Align Top"
+msgstr "Allineamento in alto"
+
+#: Layouts/Partials/AlignItemsPanel.vala:57
+msgid "Align Middle"
+msgstr "Allineamento al centro"
+
+#: Layouts/Partials/AlignItemsPanel.vala:58
+msgid "Align Bottom"
+msgstr "Allineamento in basso"
+
+#: Layouts/Partials/Artboard.vala:144 Layouts/Partials/Artboard.vala:487
+#: Layouts/Partials/Layer.vala:139 Layouts/Partials/Layer.vala:687
+msgid "Lock Layer"
+msgstr "Blocca layer"
```

Comment:
Penso che sia meglio tradurre anche layer con "Livello".

Reply:
Sì, ero un po' indeciso. "Layer" è utilizzato, comunque, in amibito tecnico al posto di "livello", no?
Se preferisci, in ogni caso, traduco come "livello" :)